react-accordion-component/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import './Accordion.css';
+
+type topics = {
```

Comment:
Please use UpperCamelCase for type names. It will make your code much easier to read.